neo/Prompt/Commands/Show.py

```diff
@@ -167,3 +172,171 @@ def execute(self, arguments=None):
 
     def command_desc(self):
         return CommandDesc('nodes', 'show connected peers')
+
+
+class CommandShowState(CommandBase):
+    def __init__(self):
+        super().__init__()
+
+    def execute(self, arguments=None):
+        height = Blockchain.Default().Height
+        headers = Blockchain.Default().HeaderHeight
+
+        diff = height - PromptData.Prompt.start_height
+        now = datetime.datetime.utcnow()
+        difftime = now - PromptData.Prompt.start_dt
+
+        mins = difftime / datetime.timedelta(minutes=1)
+        secs = mins * 60
+
+        bpm = 0
+        tps = 0
+        if diff > 0 and mins > 0:
+            bpm = diff / mins
+            tps = Blockchain.Default().TXProcessed / secs
+
+        out = "Progress: %s / %s\n" % (height, headers)
+        out += "Block-cache length %s\n" % Blockchain.Default().BlockCacheCount
+        out += "Blocks since program start %s\n" % diff
+        out += "Time elapsed %s mins\n" % mins
+        out += "Blocks per min %s \n" % bpm
+        out += "TPS: %s \n" % tps
+        print(out)
+        return out
+
+    def command_desc(self):
+        return CommandDesc('state', 'show the status of the node')
+
+
+class CommandShowNotifications(CommandBase):
+    def __init__(self):
+        super().__init__()
+
+    def execute(self, arguments):
```

Comment:
```
neo> show notifications -1
Could not parse block height can't convert negative int to unsigned

neo> show notifications bla
Could not parse block height invalid literal for int() with base 10: 'bla'

neo> show notifications
Could not execute command: object of type 'NoneType' has no len()
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/threading.py", line 884, in _bootstrap
    self._bootstrap_inner()
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/
and then some
```
unfriendly error message for an invalid arguments

similar errors are also present for `show contract 234` and `show asset 234`. I stopped reviewing at this point. Please do some manual testing, as these can be captured before PR'ing. Thanks!